integrations/terraform-mwi/provider/kubernetes_data_source_test.go

```diff
@@ -17,40 +17,78 @@ limitations under the License.
 package provider
 
 import (
+	"fmt"
+	"slices"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
-	"github.com/hashicorp/terraform-plugin-testing/knownvalue"
-	"github.com/hashicorp/terraform-plugin-testing/statecheck"
-	"github.com/hashicorp/terraform-plugin-testing/tfjsonpath"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/gravitational/teleport/lib/utils"
+	"github.com/gravitational/teleport/tool/teleport/testenv"
 )
 
 func TestAccKubernetesDataSource(t *testing.T) {
-	const config = `
+	log := utils.NewSlogLoggerForTests()
+	ctx := t.Context()
+
+	process, kubeMock := setupKubernetesHarness(t, log)
+	rootClient := testenv.MakeDefaultAuthClient(t, process)
+	_, pt := setupKubernetesAccessBot(ctx, t, rootClient)
+
+	config := fmt.Sprintf(`
 provider "teleportmwi" {
-  proxy_server = "example.com:3080"
-  join_method  = "gitlab"
-  join_token   = "example-token"
+  proxy_server = "%s"
```

Comment:
```suggestion
  proxy_server = "%[1]s"
```

I find easier to refer fmt to its indexes when the text is spread over multiple lines and the fmt arguments are too far away

Reply:
Good shout.